app/views/widget_tests/index.html.erb

```diff
@@ -1,6 +1,11 @@
 <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0" />
 
 <script>
+
+window.chatwootSettings = {
```

Comment:
Do we need to set this always?

Reply:
No, defaults will be taken if it is not set